tests/core/cycle/test_cycle_manager.py

```diff
@@ -290,3 +290,21 @@ def test_get_primary(tmpdir, cycle, current_datetime):
         )
         == 0
     )
+
+
+def test_get_scenarios_by_time(at=datetime(2024, 1, 17, 1, 0, 0, 0)):
```

Comment:
```suggestion
def test_get_scenarios_by_time():
    at_time = datetime(2024, 1, 17, 1, 0, 0, 0)
```
The `at` should not be a function parameter